lib/sisimai/bite/json/sendgrid.rb

```diff
@@ -0,0 +1,79 @@
+module Sisimai::Bite::JSON
+  # Sisimai::Bite::JSON::SendGrid parses a bounce object(JSON) which created by
+  # SendGrid. Methods in the module are called from only Sisimai::Message.
+  module SendGrid
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/JSON/SendGrid.pm
+    class << self
+      require 'sisimai/bite/json'
+
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->